ui/src/app/pages/data/concept/concept-homepage.tsx

```diff
@@ -553,8 +593,27 @@ export const ConceptHomepage = withCurrentWorkspace()(
                            expanded={this.addToSetText}
                            disable={this.activeSelectedConceptCount === 0 ||
                            !this.state.workspacePermissions.canWrite}/>
-        </FadeBox>
-      </React.Fragment>;
+        </FadeBox>}
+        {serverConfigStore.getValue().enableConceptSetSearchV2 && <React.Fragment>
+          {conceptDomainCounts.filter(domain => !selectedDomain || selectedDomain === domain.domain).map((domain) => {
+            const tabError = countsError || domainErrors.includes(domain.domain);
+            return <React.Fragment> {!tabError && !this.domainLoading(activeDomainTab) && <label style={{color: colors.accent, lineHeight: '1.5rem'}}>
+              Showing 1000 of {domain.conceptCount.toLocaleString()}</label>}
+            <ConceptTableV2 concepts={concepts}
+                            domain={activeDomainTab.domain}
+                            loading={this.domainLoading(activeDomainTab)}
+                            onSelectConcepts={this.selectConcepts.bind(this)}
+                            placeholderValue={this.noConceptsConstant}
+                            searchTerm={this.state.currentSearchString}
+                            selectedConcepts={selectedConceptDomainMap[activeDomainTab.domain]}
+                            reactKey={activeDomainTab.name}
+                            error={domainError}/>
+              {!this.domainLoading(activeDomainTab) && <Button style={{float: 'right', marginBottom: '2rem'}}
+                    disabled={this.activeSelectedConceptCount === 0 ||
+                    !this.state.workspacePermissions.canWrite}
+                    onClick={() => this.setState({conceptAddModalOpen: true})}>Finish</Button>}
```

Comment:
Sorry, I missed this the first time. Not sure how closely we need to mirror CB at this point, but should this be a 'Finish & Review` button that opens the sidebar, then have a 'Save Concepts' button at the bottom of the selected concepts list?

Reply:
Good point

Reply:
Done